src/pages/Products/Products.js

```diff
@@ -1,7 +1,110 @@
 import React from 'react';
+import './Products.scss';
 
 function Products() {
-  return <p>Hi</p>;
+  return (
+    <div className="products">
+      <div className="bannerImg">
+        <img src="/images/products/SPP_DT_top_banner_LBM_1440x112.png" />
+      </div>
+      <div className="info">
+        <div className="layoutInfoFull">
+          <div className="categoryWrap">
+            <ol className="category">
+              <li className="cateList">
+                <a href="/" title="홈">
+                  <p className="home">홈</p>
+                </a>
```

Comment:
```suggestion
                <Link to="/" title="홈" className="home">
                 홈
                </Link>
```
불필요한 depth없이 위와 같은 방식으로 구현할 수 있겠습니다~!